frontend/public/components/image-stream.tsx

```diff
@@ -4,7 +4,7 @@ import * as _ from 'lodash-es';
 // eslint-disable-next-line no-unused-vars
 import { K8sResourceKindReference } from '../module/k8s';
 import { ColHead, DetailsPage, List, ListHeader, ListPage } from './factory';
```

Comment:
@spadgett mentioned `<Overflow>` vs `<OverflowYFade>`.  I'm not sure what the other component does, but the screenshots look good (I didn't pull & run this branch).

Reply:
Both `Overflow` and `OverflowYFade` visually do the same, but `Overflow` adds an onClick handler that selects the text.

Reply:
Wouldn't have guessed that by the names :)